pkg/pool-weighted/test/ManagedPoolSettings.test.ts

```diff
@@ -826,7 +826,13 @@ describe('ManagedPoolSettings', function () {
   });
 
   describe('circuit breakers', () => {
-    async function getBptPrice(): Promise<BigNumber> {
+    async function getUnscaledBptPrice(): Promise<BigNumber> {
+      const totalSupply = await pool.getActualSupply();
+
+      return fpDiv(fpMul(totalSupply, poolWeights[0]), initialBalances[0]);
```

Comment:
The first token has 18 decimals, so we're actually not testing anything

Reply:
For now, making it use a dynamic tokenIndex.